actionpack/lib/system_test_case.rb

```diff
@@ -0,0 +1,57 @@
+require "system_testing/test_helper"
+require "system_testing/driver_adapter"
+
+module Rails
```

Comment:
Ah, I missed that. Then going with ActionDispatch::SystemTest probably makes sense. But agree no sense in making any changes until we really know if we love that name!